src/main/webapp/webapptests/spec/SpecSearch.js

```diff
@@ -48,31 +50,38 @@ describe("Search", function() {
         var listContainer = document.createElement("div");
         listContainer.id = "lists-container";
 
+        var listsLabel = document.createElement("p");
+        listsLabel.id = "num-lists-results"
+
         var booksLabel = document.createElement("p");
         booksLabel.id = "num-book-results";
 
         var bookContainer = document.createElement("div");
         bookContainer.id = "books-container";
 
+        var panel = document.createElement("div");
+
         var mockWindow = {location: {href: "search-results.html?topic=math", search: "?topic=math"}};
 
         it("should create result elements inside tutorContainer, listContainer, and bookContainer", async function() {
-            spyOn(window, "fetch").and.returnValues(Promise.resolve({json: () => Promise.resolve(tutors)}), Promise.resolve({json: () => Promise.resolve(lists)}), Promise.resolve({json: () => Promise.resolve(books)}));
+            spyOn(window, "fetch").and.returnValues(Promise.resolve({json: () => Promise.resolve(tutors)}), 
+                                                    Promise.resolve({json: () => Promise.resolve(lists)}), 
+                                                    Promise.resolve({json: () => Promise.resolve(books)}));
```

Comment:
Isn't it possible to return the specific json to their specif fetch calls only?

Reply:
I tried using spyOn(window, "fetch").withArgs() but that didn't seem to work. Is there another way?

Reply:
Yeah, my guess would be to specify the args but if that didn't work I'm not sure what to do. If this approach works, I think we can just stay with it